pkgs/development/compilers/llvm/common/lldb.nix

```
@@ -16,18 +16,32 @@
 , darwin
 , lit
 , makeWrapper
-, src
 , lua5_3
 , ninja
+, runCommand
+, src ? null
+, monorepoSrc ? null
 , patches ? [ ]
 , enableManpages ? false
 }:
 
+let
+  src' =
+    if lib.isDerivation monorepoSrc then
```

Could be a variety of things that can be coerced to a string in string interpolation, so this is better.

```suggestion
    if monorepoSrc != null then
```